src/components/Sorted.jsx

```diff
@@ -0,0 +1,50 @@
+import { useSearchParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchSortOrder } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function Sorted () {
+
+    const [searchParams, setSearchParams] = useSearchParams();
+    const [sorted, setSorted] = useState([]);
+
+    const [isLoading, setIsLoading] = useState(true);
+
+    const sort = searchParams.get("sort_by");
+    const order = searchParams.get("order_by");
+
+    useEffect(()=>{
+        fetchSortOrder(sort, order)
+          .then((articles) => {
+            setSorted(articles);
```

Comment:
it looks as though you're not setting the articles here but rather an object with the key of articles?